migrations/2_deploy_contracts.js

```diff
@@ -382,7 +398,6 @@ module.exports = function(deployer, network, accounts) {
         .then((securityTokenRegistryProxy) => {
```

Comment:
Shouldn't we call `setProtocolFactory()` and `setLatestVersion()` in the migrations

Reply:
Done